tests/Utils/ValidatorTest.php

```diff
@@ -16,89 +16,87 @@
 
 class ValidatorTest extends TestCase
 {
-    private $object;
+    private $validator;
 
-    public function __construct()
+    protected function setUp(): void
     {
-        parent::__construct();
```

Comment:
caused

```
Other deprecation notices (1)

  1x: The "PHPUnit\Framework\TestCase::__construct()" method is considered internal This method is not covered by the backward compatibility promise for PHPUnit. It may change without further notice. You should not extend it from "App\Tests\Utils\ValidatorTest".